docs/metadata.md

```diff
@@ -18,12 +14,12 @@ The Alexandria Metadata Convention contains 4 categories of structured metadata
 
 Examples: disease, species, organ
 
-To allow query at varying levels of specificities, the Alexandria Metadata Convention uses ontology-valued metadata whenever possible. These ontologies are tree-structured heirarchies curated by experts and maintained by EBI. For the purposes of the Alexandria Metadata Convention, please select the most specific value in the heirarchy as possible. For use of this metadata during query of the Alexandria database, a query result will include all child values of the query term. Single Cell Portal treats these values as group, or categorical variables. 
+To allow query at varying levels of specificities, the Alexandria Metadata Convention uses ontology-valued metadata whenever possible. These ontologies are tree-structured hierarchies curated by experts and easily accessible through a [Lookup Service](https://www.ebi.ac.uk/ols/index) maintained by EBI. For the purposes of the Alexandria Metadata Convention, please select the most specific value in the hierarchy as possible. For use of this metadata during query of the Alexandria database, a query option can allow the inclusion of all child values of the query term. Single Cell Portal treats these values as group, or categorical variables. 
```

Comment:
Note "query option can allow the inclusion of all child values of the query term" is not yet an implemented feature for Alexandria. Can we decide on a phrasing to indicate that the functionality is on the roadmap but not yet available?

Reply:
"In an future update, querying this metadata on Alexandria will allow the inclusion of all its child values."? My understanding might be incomplete for this one

Reply:
Would it be acceptable to remove "For use of this metadata during query of the Alexandria database, a query option can allow the inclusion of all child values of the query term." and add something along this lines back to the documentation when the feature is available? It might be confusing to refer to functionality that has not yet been implemented.